docs/deploy-custom-image.rst

```diff
@@ -566,3 +566,77 @@ If you are using a private repository to push your images to, make sure to pass
 Or pass it to :ref:`upload <commands-upload>` command::
 
     $ shub image upload --username johndoe --password yourpass
+
+
+Container works locally but fails in scrapy cloud
+-------------------------------------------------
+
+Prior to running ``start-crawl`` in Scrapy Cloud, some configurations
+are set to ensure we can run an isolated process.
+This can lead to issues that are quite hard to debug and find the
+root cause.
+To aid in this process, below you willl find some steps that
+are quite similar to what actually runs in scrapy cloud.
+
+Run your container in interactive with ``bash`` (or any other
+terminal that is available). Please replace the 2 occurrences of
+``<SPIDER-NAME>`` with the actual spider that is to run::
+
+    $ docker run \
+    -it \
+    -e SHUB_JOBKEY=123/4/5 \
+    -e SHUB_JOB_DATA='{
+        "_shub_worker": "kumo",
+        "api_url": "https://app.zyte.com/api/",
+        "auth": "<AN AUTH KEY>",
```

Comment:
This one is not mentioned in the paragraph above, and style is inconsistent with `<SPIDER-NAME>` (maybe `AUTH-KEY`?)

Reply:
This one is not required, I update to some constant